### ISSUE TITLE
SA1101 is not raised when implementing property with an expression

### ISSUE THREAD

**Issue by vweijsters** [DotNetAnalyzers/StyleCopAnalyzers#2845]
The following code does not raise SA1101 where it should have:

```C#
internal sealed class UnknownElementTypeSyntaxList : SeparatedSyntaxListWrapper<TNode>
{
    private readonly object underlyingList;

    //// ...

    public override object UnderlyingList => underlyingList;

    //// ...
}
```

the definition of the property is:

```C#
internal abstract class SeparatedSyntaxListWrapper<TNode> : IEquatable<SeparatedSyntaxListWrapper<TNode>>, IReadOnlyList<TNode>
{
    //// ...

    [EditorBrowsable(EditorBrowsableState.Never)]
    public abstract object UnderlyingList
    {
        get;
    }

    //// ...
}
```
